packages/graphql/tests/tck/tck-test-files/directives/auth/arguments/roles-where.md

```diff
@@ -0,0 +1,1774 @@
+# Cypher Auth Where
```

Comment:
Markdown-based tck tests have been discontinued in favor of jest-based tests, migration should be pretty straightforward (an example can be found in `packages/graphql/tests/tck/tck-test-files/directives/auth/arguments/where/where.test.ts`